### PR TITLE
[14.0][MIG]osi_analytic_segments_stock_scrap

### PR DESCRIPTION
**Pull request by hpatelserpentcs**
Migrated the module osi_analytic_segments_stock_scrap in v14.

### CODE REVIEW

File: osi_analytic_segments_stock_scrap/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    "summary": "Provide a user-defined list of scrapping reasons",
+    "version": "14.0.1.0.0",
+    "license": "AGPL-3",
+    "author": "Open Source Integrators",
+    "maintainer": "Open Source Integrators",
+    "website": "http://www.opensourceintegrators.com",
+    "category": "Inventory",
+    "depends": [
+        "stock_analytic",
+        "osi_scrap_reason_code",
```
Comment (patrickrwilson) [inline osi_analytic_segments_stock_scrap/__manifest__.py]: scrap reason code was moved to OCA which we should use. The module does work with using the OCA module instead so just changing this line is all that's needed.

**Review by patrickrwilson — CHANGES_REQUESTED** [review]
OSI scrap reason code has moved to OCA, please update dependency.

**Review by patrickrwilson — APPROVED** [review]
Tested and LGTM

**Review by smangukiya — APPROVED** [review]
LGTM.

**Pull request merged**